vllm/model_executor/models/llama.py

```diff
@@ -319,6 +319,14 @@ class LlamaForCausalLM(nn.Module):
         "lm_head": "output_embeddings",
     }
     embedding_padding_modules = ["lm_head"]
+    bitsandbytes_stacked_params_mapping = {
+        # shard_name, weight_name, index
+        "q_proj": ("qkv_proj", 0),
+        "k_proj": ("qkv_proj", 1),
+        "v_proj": ("qkv_proj", 2),
+        "gate_proj": ("gate_up_proj", 0),
+        "up_proj": ("gate_up_proj", 1),
+    }
```

Comment:
Does this mean only Llama is supported at the moment and we will need to add this mapping for each model? It is worth noting this needs to be present for each model - maybe the bitsandbytes.py file is a good place to put this information

Reply:
yes, only llama is supported right now. Similar to the feature of LoRA, some changes need to make to each model to support QLoRA. Different models might have slightly different list of module names. 

This variable also serves as a mark to indicate whether QLoRA is supported in the model. It is following the same pattern of LoRA, which use the variable of supported_lora_modules for indication.